java/spark/src/test/java/com/lancedb/lance/spark/read/SparkConnectorReadTest.java

```diff
@@ -185,4 +185,23 @@ public void supportBroadcastJoin() {
     assertEquals(1, desc.size());
     assertTrue(desc.get(0).getString(0).contains("BroadcastHashJoin"));
   }
+
+  @Test
+  void supportReadOptions() {
+    Dataset<Row> df =
+        spark
+            .read()
+            .format("lance")
+            .option("version", "1")
+            .load(LanceConfig.getDatasetUri(dbPath, TestUtils.TestTable1Config.datasetName));
+    assertEquals(2, df.count());
```

Comment:
Can we add some description of why there is `2`? The first version, wrote 2 records?